src/utils.cc

```diff
@@ -499,7 +506,11 @@ std::string GetDefaultResourceDirectory() {
     result = resource_directory;
   }
 
-  return NormalizePath(result);
+  result = NormalizePath(result);
+#if defined(_WIN32)
+  result = std::regex_replace(result, std::regex("/"), "\\");
```

Comment:
NormalizePath should do this.

Reply:
NormalizePath does the other way around. And I'm suspicious that clang need the "\" form. Can you stop all the earlier appveyor builds to see how this work?